observers/base_pitch/base_pitch.py

```diff
@@ -155,4 +155,4 @@ def compute_base_pitch_from_imu(
         rotation_ars_to_world @ rotation_imu_to_ars @ rotation_base_to_imu
     )
     pitch_base_in_world = compute_pitch_frame_in_parent(rotation_base_to_world)
-    return pitch_base_in_world
```

Comment:
Woah, good catch :fishing_pole_and_fish: 

Thank you for pointing this out. Here is some documentation around this: https://github.com/tasts-robots/upkie/issues/74

Let's fix the signs in a separate PR.